python/lsst/pipe/analysis/utils.py

```diff
@@ -38,15 +40,15 @@
            "lineFromP2Coeffs", "linesFromP2P1Coeffs", "makeEqnStr", "catColors", "setAliasMaps"]
 
 
-def writeParquet(table, path, badArray=None):
-    """Write an afwTable into Parquet format
+def writeParquet(dataRef, table, badArray=None):
+    """Writes an afwTable to a desired ParquetTable butler dataset
```

Comment:
From https://developer.lsst.io/python/numpydoc.html#writing-summaries-for-functions-and-methods \:
"For functions and methods, write in the imperative voice. That is, the summary is treated as a command that the API consumer can give." (i.e. Write -- no "s" -- was correct)